### DISCUSSION TITLE
allow to reset form with new values

### DISCUSSION THREAD

**Discussion by schiller-manuel** [TanStack/form#596] [Ideas] [answered]
After form submission I want to keep the form open and make the form pristine again, but don't want to lose the current values.

Inspiration: In RHF this is possible via calling `reset` with new values: https://www.react-hook-form.com/api/useform/reset/

**Comment by crutchcorn** [comment, marked as answer]
Closing, as I'm moving this to an issue to track:

https://github.com/TanStack/form/issues/785

This would be a good first issue if you wanted to make a PR :)
